kafka/source/pkg/adapter/adapter.go

```diff
@@ -91,6 +92,11 @@ func (a *Adapter) Start(stopCh <-chan struct{}) error {
 	if err != nil {
 		return fmt.Errorf("failed to create the config: %w", err)
 	}
+	config.Consumer.Offsets.AutoCommit = struct {
```

Comment:
I recall this is the default config right?

sarama/config.go:459

```
	c.Consumer.Offsets.AutoCommit.Enable = true
	c.Consumer.Offsets.AutoCommit.Interval = 1 * time.Second
```

Reply:
Ah ok enable false i see, you don't need to create the struct like that, you can simply do:

```
c.Consumer.Offsets.AutoCommit.Enable = false
```

Reply:
Let's wait for test results and then I'll clean up.

Reply:
Done